quotamodel.c

```diff
@@ -480,6 +481,8 @@ DiskQuotaShmemSize(void)
 	size = sizeof(ExtensionDDLMessage);
 	size = add_size(size, hash_estimate_size(MAX_DISK_QUOTA_BLACK_ENTRIES, sizeof(GlobalBlackMapEntry)));
 	size = add_size(size, hash_estimate_size(diskquota_max_active_tables, sizeof(DiskQuotaActiveTableEntry)));
+	size = add_size(size, hash_estimate_size(diskquota_max_active_tables, sizeof(DiskQuotaRelationEntry)));
+	size = add_size(size, hash_estimate_size(diskquota_max_active_tables, sizeof(DiskQuotaPgClassCacheEntry)));
```

Comment:
```suggestion
	size = add_size(size, hash_estimate_size(diskquota_max_active_tables, sizeof(DiskQuotaPgClassCacheEntry)));
	size = add_size(size, hash_estimate_size(diskquota_max_active_tables, sizeof(DiskQuotaRelidCacheEntry)));
```

It looks that we forget to allocate shared memory for `relid_cache`?

Reply:
Yes, it is my mistake.